helm/examples/logging/README.md

```diff
@@ -100,7 +100,7 @@ The available log data columns include traceid(TRC), spanid(SPN), messageid(MID)
     helm install myhpcc ./hpcc --set global.logging.fields="STD-JOB"
     
     Example: Include trace and span info in addition to the standard columns:
-    helm install myhpcc ./hpcc --set global.logging.fields="STD-JOB"
+    helm install myhpcc ./hpcc --set global.logging.fields="STD+TRC+SPN"
```

Comment:
Would render better inside a "code fence" 
```bash
helm install myhpcc ./hpcc --set global.logging.fields="STD+TRC+SPN"
```

Reply:
Perhaps, but the style isn't changed here, only the content.